testdata/manifest/error3.toml

```diff
@@ -0,0 +1,9 @@
+ignored = ["github.com/foo/bar"]
+
+[[override]]
+  name = "github.com/golang/dep/internal/gps"
```

Comment:
I realize there are probably some of these hanging around from when we migrated gps into dep, but the names have to point to project roots. We're adding validation for this soon - better to have them correct now than create more problems to fix later.